src/JustSaying.Extensions.DependencyInjection.Microsoft/IServiceCollectionExtensions.cs

```diff
@@ -45,28 +45,28 @@ public static IServiceCollection AddJustSaying(this IServiceCollection services)
         /// Adds JustSaying services to the service collection.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/> to add JustSaying services to.</param>
-        /// <param name="regions">The AWS region(s) to configure.</param>
+        /// <param name="region">The AWS region to configure.</param>
         /// <returns>
         /// The <see cref="IServiceCollection"/> specified by <paramref name="services"/>.
         /// </returns>
         /// <exception cref="ArgumentNullException">
-        /// <paramref name="services"/> or <paramref name="regions"/> is <see langword="null"/>.
+        /// <paramref name="services"/> or <paramref name="region"/> is <see langword="null"/>.
         /// </exception>
-        public static IServiceCollection AddJustSaying(this IServiceCollection services, params string[] regions)
+        public static IServiceCollection AddJustSaying(this IServiceCollection services, string region)
         {
             if (services == null)
             {
                 throw new ArgumentNullException(nameof(services));
             }
 
-            if (regions == null)
+            if (string.IsNullOrWhiteSpace(region))
             {
-                throw new ArgumentNullException(nameof(regions));
+                throw new ArgumentNullException(nameof(region));
```

Comment:
Let's change to just `ArgumentException` to cover the non-null cases.

Reply:
Sounds good, done 👍